### DISCUSSION TITLE
闫永强老师的yolov5手势识别表情包的数据集还有吗？怎么现在找不到了，有同志可以再分享一下吗？

### DISCUSSION THREAD

**Discussion by 2677269950** [datawhalechina/DOPMC#160] [问题求助] [answered]
https://mp.weixin.qq.com/s?__biz=MzIyNDA1NjA1NQ==&mid=2651029907&idx=1&sn=ef75c710e7fbcc961b83307af2f43f72&scene=21#wechat_redirect
是这个学习内容

**Comment by szuRyan** [comment, marked as answer]
链接：https://pan.baidu.com/s/1GkKhLnSItHoVuvc6AjRCRQ 
提取码：wqsa

**Comment by 2677269950** [comment]
您好，这个手势识别您复现了吗？






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "datawhalechina/DOPMC"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年9月14日(星期四) 中午11:08
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [datawhalechina/DOPMC] 闫永强老师的yolov5手势识别表情包的数据集还有吗？怎么现在找不到了，有同志可以再分享一下吗？ (Discussion #160)





 
链接：https://pan.baidu.com/s/1GkKhLnSItHoVuvc6AjRCRQ
 提取码：wqsa

**Comment by szuRyan** [comment]
从数据 模型 到整体的c++推理实现都是我做的，实现效果是demo效果，真正用起来还有很多需要优化

**Comment by 2677269950** [comment]
您好，那请问还有源码吗？我后面c++实现那一块没有复现成功，前面的我自己找了数据集，然后标注了，做到转ncnn那一步，后面使用vsuial studio2022复现c++那一块就报错，所以想请教您一下


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "datawhalechina/DOPMC"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年9月14日(星期四) 中午11:12
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [datawhalechina/DOPMC] 闫永强老师的yolov5手势识别表情包的数据集还有吗？怎么现在找不到了，有同志可以再分享一下吗？ (Discussion #160)





 
从数据 模型 到整体的c++推理实现都是我做的，实现效果是demo效果，真正用起来还有很多需要优化

**Comment by szuRyan** [comment]
公众号的文章已经带了全部的源码，具体问题可以贴出来看看是什么问题

**Comment by 2677269950** [comment]
这一步，我就没看到有过多的不支持层，然后后面的c++实现：

最后，能加您个联系方式吗？


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "datawhalechina/DOPMC"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年9月14日(星期四) 中午11:16
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [datawhalechina/DOPMC] 闫永强老师的yolov5手势识别表情包的数据集还有吗？怎么现在找不到了，有同志可以再分享一下吗？ (Discussion #160)





 
公众号的文章已经带了全部的源码，具体问题可以贴出来看看是什么问题

**Comment by szuRyan** [comment]
你这个没有显示错误的信息，你是发的图片？你这个数字是你的qq号？
